composer.json

```
@@ -11,16 +11,19 @@
     ],
     "require": {
         "php": ">=7.0",
+        "ext-json": "*",
         "elasticsearch/elasticsearch": "^5.1.0||^6.0.0"
     },
     "require-dev": {
         "atoum/atoum":                      "^2.8||^3.0",
         "m6web/coke":                       "~1.2.0",
         "m6web/symfony2-coding-standard":   "~1.2.0",
-        "symfony/dependency-injection":     "^2.3||^3.0",
-        "symfony/event-dispatcher":         "^2.3||^3.0",
-        "symfony/config":                   "^2.3||^3.0",
-        "symfony/yaml":                     "^2.3||^3.0"
+        "symfony/dependency-injection":     "^5.0",
+        "symfony/event-dispatcher":         "^5.0",
+        "symfony/config":                   "^5.0",
+        "symfony/yaml":                     "^5.0",
+        "symfony/http-foundation":          "^5.0",
+        "symfony/http-kernel":              "^5.0"
```

```suggestion
        "symfony/dependency-injection":     "^4.4||^5.0",
        "symfony/event-dispatcher":         "^4.4||^5.0",
        "symfony/config":                   "^4.4||^5.0",
        "symfony/yaml":                     "^4.4||^5.0",
        "symfony/http-foundation":          "^4.4||^5.0",
        "symfony/http-kernel":              "^4.4||^5.0"
```

because we can't be that violent ("5.0" is like "4.4 without deprecations" so we can easily allow it here)